go/noise/gaussian_noise.go

```diff
@@ -155,6 +186,23 @@ func addGaussian(x, sigma float64) float64 {
 	return roundToMultipleOfPowerOfTwo(x, granularity) + float64(sample)*granularity
 }
 
+// getConfidenceIntervalGaussian computes the confidence interval using the inverse error function.
+func getConfidenceIntervalGaussian(noisedValue, confidenceLevel, sigma float64) ConfidenceInterval {
+	shiftingValue := inverseCDFGaussian(sigma, confidenceLevel)
```

Comment:
I am not sure I like the name "shiftingValue". I also don't have a good alternative suggestion. Maybe something short like "z" works here. I think it makes sense to be conistent with the Lapalce implemntation here.

Reply:
That makes sense, thank you. I will switch it to z. :)